scripts/install-jstz-cli.sh

```diff
@@ -31,13 +31,11 @@ jstz_download() {
 }
 
 jstz_configure() {
-    if [ -d "$jstz_home" ]; then
-        echo "Configuration already exists. Skipping..."
-    else
-        echo -n "Configuring jstz..."
+    echo -n "Configuring jstz..."
 
-        mkdir -p "$jstz_home"
-        cat >"$jstz_home/config.json" << EOF
+    mkdir -p "$jstz_home"
+    
+    cat >"$jstz_home/config.json" << EOF
```

Comment:
This overwrites the config for each installation -- I think we should provide a way for the user to keep their config and just update the weeklynet endpoint (using sed).

...Though now I think about it, the config will essentially be useless after the update 🤔